core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -1939,24 +1941,24 @@ private[spark] class DAGScheduler(
       hostToUnregisterOutputs: Option[String],
       maybeEpoch: Option[Long] = None): Unit = {
     val currentEpoch = maybeEpoch.getOrElse(mapOutputTracker.getEpoch)
+    logDebug(s"Considering removal of executor $execId; " +
+      s"fileLost: $fileLost, currentEpoch: $currentEpoch")
     if (!failedEpoch.contains(execId) || failedEpoch(execId) < currentEpoch) {
       failedEpoch(execId) = currentEpoch
-      logInfo("Executor lost: %s (epoch %d)".format(execId, currentEpoch))
+      logInfo(s"Executor lost: $execId (epoch $currentEpoch)")
```

Comment:
If you don't mind, can we avoid a style change like this in order to focus the feature?
```
- logInfo("Executor lost: %s (epoch %d)".format(execId, currentEpoch))
+ logInfo(s"Executor lost: $execId (epoch $currentEpoch)")
```

Reply:
@attilapiros requested this change. I'm fine with either style. Both styles are used in the file.
I can change it back, but I don't want to keep changing back and forth.
So let me do that only if you are satisfied with the substantive part of this change and are prepared to approve it.

Reply:
Guilty as charged. My thinking was the following: as the string interpolation preferred in the project and the change itself is quite tiny and has zero risk and we are here and touching these lines it would be good to do it now (aka Boy Scout Rule).

Reply:
while I don't think this is a big deal either way, I also try to follow the "boy scout rule" as long as its fixing stuff very close to the actual change.

Reply:
@dongjoon-hyun based on Attila's and Imran's comments, I'm inclined to stick with the string interpolation here and below, since it is all within the method I'm actively modifying.
Are you ok with resolving this then?